electron-app/src/server/websites/mastodon/mastodon.service.ts

```diff
@@ -124,6 +131,7 @@ export class Mastodon extends Website {
     file: PostFile,
     altText: string,
   ): Promise<{ id: string }> {
+    this.logger.debug("Mastodon uploadMedia")
```

Comment:
q: purpose?

Reply:
Was part of my tracing how everything hung together; removed.